hyrule_field.logic

```diff
@@ -1,11 +1,63 @@
 area HyruleField:
-  room EastHillsNorth
-  room EastHillsMid
-  room EastHillsSouth
+  room EastHillsNorth:
+    node West:
+      door West
+      ennemy WestAcroBandits
+      door South
+    node WestFusionChest:
+      chest WestFusionChest
+    node GoldenRopeFusion:
+      ennemy GoldenRope
+    node MainEast:
+      door FarmersHouse
+      door SouthEast
+      ennemy EastAcroBandits
+      ennemy Octorocs
```

Comment:
I see no reason to split this group of enemies. Also, same typo
```suggestion
      enemy EastAcroBanditsAndOctorocs
```

Reply:
Enemizer can always still look inside the groups - it doesn't need to be split in the logic for that.